approval_frame/forms.py

```diff
@@ -17,3 +18,22 @@ class RegistrationFormCustom(RegistrationFormUniqueEmail):
         error_messages={'invalid': _("This value may contain only letters, "
             + "numbers and ./+/-/_ characters.")}
     )
+
+class PasswordResetFormCustom(PasswordResetForm):
```

Comment:
Sure @ashimaathri , that sounds good. I have removed this validation in the latest commit. Let me know if it is satisfactory. Thanks!